cluster-autoscaler/simulator/delegating_shared_lister.go

```diff
@@ -18,6 +18,7 @@ package simulator
 
 import (
 	"fmt"
+
```

Comment:
This is obviously trivial so I'm ok with it, but in future I think it's better to avoid mixing changes in cloudprovider/ with changes in any other parts of CA. The rationale is that cloudprovider and the rest of CA usually have a different set of owners and the changes often need to be reviewed by a different set of people.
Unless the changes are strictly connected it's better to split them into 2 PRs that can be reviewed by owners of the given piece of code.

Reply:
Sure. Thanks for the note.